pkgs/development/python-modules/nbconflux/default.nix

```diff
@@ -2,26 +2,33 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
-  isPy27,
   nbconvert,
   pytestCheckHook,
+  pythonOlder,
   requests,
   responses,
+  setuptools,
+  versioneer,
 }:
 
 buildPythonPackage rec {
   pname = "nbconflux";
   version = "0.7.0";
   format = "setuptools";
-  disabled = isPy27; # no longer compatible with python 2 urllib
+  disabled = pythonOlder "3.2"; # SafeConfigParser was deprecated here
```

Comment:
This is now unnecessary and can be dropped, because `versioneer` encodes its requirements (well, it would be unnecessary in general because we don’t package a Python 3.2 implementation anyway, though).

LGTM other than that!

Reply:
Doooooone!